src/pages/CarsList/CarsList.tsx

```diff
@@ -0,0 +1,12 @@
+import CarListComponent from '../../components/CarListComponent/CarListComponent';
+import { Container } from './CarsListStyled';
+
+const CarList = () => {
+  return (
+    <Container>
+      <CarListComponent />

```

Comment:
This component doesn't seem to add much value. You should probably just merge these two ditch the 'Component' in the name.